GeosCore/tomas_mod.F90

```diff
@@ -8568,7 +8721,9 @@ SUBROUTINE SUBGRIDCOAG( NDISTINIT, NDIST, MDIST, BOXVOLUME,TEMP, &
 
     ! determine the number of new particles left after coagulation
     do k=1,ibins
-       ndistfinal(k)=ndistinit(k)*exp(-kcoag(k)*tscale)
+       !ndistfinal(k)=ndistinit(k)*exp(-kcoag(k)*tscale)
+       !print*, 'Betty is here tscale ',tscale,k,kcoag(k)
```

Comment:
Please remove debug code

Reply:
It would be fine to remove these 2 lines that are commented out here.

Reply:
I will leave the preference on syntax up to you. We can keep it as is in this pull request.